src/Updater.php

```diff
@@ -87,6 +87,13 @@ public function __construct(ConfigFactoryInterface $config_factory, ModuleInstal
     $this->eventDispatcher = $event_dispatcher;
   }
 
+  /**
+   * State of last update execution.
+   *
+   * @var bool
+   */
+  protected $successfulExecute = FALSE;
```

Comment:
Here the default is FALSE but when we start an update we default it to TRUE. That feels super odd.